tensorboard/webapp/widgets/data_table/data_table_component.ng.html

```diff
@@ -20,7 +20,25 @@
           <!-- This header is intentionally left blank for the color column -->
         </th>
         <ng-container *ngFor="let header of headers;">
-          <th>{{ getHeaderTextColumn(header) }}</th>
+          <th [ngSwitch]="header">
+            <div *ngSwitchCase="ColumnHeaders.VALUE_CHANGE" class="cell">
+              <mat-icon
+                class="delta-arrow"
+                svgIcon="change_history_24px"
+              ></mat-icon>
+              <div>{{ getHeaderTextColumn(header) }}</div>
+            </div>
+            <div *ngSwitchCase="ColumnHeaders.PERCENTAGE_CHANGE" class="cell">
+              <mat-icon
+                class="delta-arrow"
+                svgIcon="change_history_24px"
+              ></mat-icon>
+              <div>{{ getHeaderTextColumn(header) }}</div>
+            </div>
+            <div *ngSwitchDefault class="cell without-icon">
+              {{ getHeaderTextColumn(header) }}
+            </div>
+          </th>
```

Comment:
Can we move the header switch logic to only icon?
For example
```
          <th [ngSwitch]="header" class="cell">
              <mat-icon
                *ngSwitchCase="ColumnHeaders.VALUE_CHANGE"
                class="delta-arrow"
                svgIcon="change_history_24px"
              ></mat-icon>
              <mat-icon
                *ngSwitchCase="ColumnHeaders.PERCENTAGE_CHANGE"
                class="delta-arrow"
                svgIcon="change_history_24px"
              ></mat-icon>
              <div
                *ngSwitchDefault
                class="without-icon"
              >
              </div>
              <div>{{ getHeaderTextColumn(header) }}</div>
          </th>
```

I tested it a bit and it looks similar

Reply:
Same, I would argue for the runData below. But that's not included in this PR.

Reply:
Except color (which you've handled differently) I don't feel anything might \not contain header text. Let's just make the switch cases for icons. (From my perspective, "no icon" is also one of the icon options)